src/components/chip-set/chip-set.scss

```diff
@@ -167,11 +162,21 @@ limel-icon.mdc-chip__icon.mdc-chip__icon--leading {
 }
 
 .mdc-floating-label {
+    @include looks-like-input-label;
     &.mdc-text-field--required::after {
         content: '*';
     }
-    &.force-float {
+    &.lime-floating-label--float-above,
+    &.lime-floating-label--float-above.mdc-floating-label--float-above {
         @include label--float-above;
+        top: $floating-label-top-value;
+    }
+}
+
+.lime-notched-outline--notched {
+    .mdc-notched-outline__notch {
```

Comment:
The width of the notch isn't consistent. For the label "Animal", it's a good bit too large when the picker is empty and focused, perfect when the picker has content and is focused, and just a smidge too when when it has content and is not focused.

Example:

![image](https://user-images.githubusercontent.com/154725/101766388-00c79880-3ae3-11eb-912d-fe0024c73b92.png)

I noticed that the notch size depends on where on the component you click (at least in the case recorded below). If you click on the edge of the component the notch becomes the correct size 😮

![limel-chip-set notch](https://user-images.githubusercontent.com/154725/101767322-56507500-3ae4-11eb-88fd-6fa8621498d2.gif)

NOTE! I do _not_ think we should fix this here. It's a small visual artefact, and I think we should get this PR working and merged before spending time polishing out such tiny bumps.

Reply:
This is also a custom solution from us. We shouldn't fix it as you say. Not before we've upgraded MD and verified that the other cut-off labels won't be cut-off anymore. These problems have the same source. (some scripts that adds inline `width` to the containing div)